fmriprep/run_workflow.py

```diff
@@ -82,7 +82,8 @@ def create_workflow(opts):
         'debug': opts.debug,
         'skull_strip_ants': opts.skull_strip_ants,
         'output_dir': op.abspath(opts.output_dir),
-        'work_dir': op.abspath(opts.work_dir)
+        'work_dir': op.abspath(opts.work_dir),
+        'remove_unnecessary_outputs': False
```

Comment:
I would honestly prefer to have this in your ~/.nipype/nipype.cfg, but whatever you want :). Especially bc you are now connecting the outputs corresponding to the generated reports into a subsequent node. This options will not have any effect in practice.